helm_deploy/whereabouts-api/templates/deployment.yaml

```diff
@@ -34,7 +34,7 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /health
+              path: /ping
```

Comment:
once service is up stop other services failing taking down this service (cascading failures)